src/extensions/core/nodeTemplates.ts

```diff
@@ -348,110 +350,109 @@ class ManageTemplates extends ComfyDialog {
   }
 }
 
-app.registerExtension({
+const manage = new ManageTemplates()
+
+// @ts-expect-error fixme ts strict error
```

Comment:
For any of these suppressions, if you know the expected type here, could you add it and remove the error suppression?

Reply:
(Unless it's a lot of effort, then we can punt to when we do a more comprehensive fix)